docs/DeveloperGuide.md

```diff
@@ -290,31 +290,32 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC2 - List all people saved in `BizBook`**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to list all people saved in their `BizBook`.
```

Comment:
Consider adding a "by entering a command" here since it's only later mentioned that this is done through a command in `1b`